test/built-ins/Map/groupBy/string.js

```diff
@@ -11,12 +11,12 @@ includes: [compareArray.js]
 features: [array-grouping, Map]
 ---*/
 
-const string = 'abcd';
+const string = '🥰💩🙏😈';
```

Comment:
oh i didn't mean _just_ surrogate pairs :-) this is fine, but i was expecting a mix of code units and code points.